nautobot/core/views/mixins.py

```diff
@@ -903,3 +914,33 @@ def perform_bulk_update(self, request, **kwargs):  # pylint: disable=arguments-d
             return redirect(self.get_return_url(request))
         data.update({"table": table})
         return Response(data)
+
+
+class ObjectChangeLogViewMixin(NautobotViewSetMixin):
+    """
+    UI mixin to list a model's changelog queryset
+    """
+
+    base_template = None
+
+    def changelog(self, request, *args, **kwargs):
+        data = {
+            "base_template": self.base_template,
+            "template": "extras/object_changelog.html",
```

Comment:
Not blocking and def a nitpick, but can we use the generic `get_template_name()` here instead of hardcoding the template name?

Reply:
Because we are essentially making `changelog` and `notes` an "action" for NautobotUIViewSets I disagree: https://github.com/nautobot/nautobot/blob/develop/nautobot/core/views/mixins.py#L316

`get_template_name` should know about these templates and we should align them with the template patterns we've set forth with NautobotUIViewSets.